Automattic/wp-calypso
# [Odie] Apply new help center redesign for Wapuu

Related to: https://github.com/Automattic/ai-services/issues/555

This set of changes are intended to implement @mattrwalker changes for the Help Center integration with Odie. This is still a work in progress chain of Pull Requests, follow up pull requests will follow.

## Proposed Changes

- Added avatars for the conversation
- Moved the Odie code in Help-Center content to its own component
- Added new look and feel for the chat messages
- Added new look and feel for the input messages
- Added new button to jump to the most recent messages
- Added EllipsisMenu in the header with options (new chat, send transcript)
- Added new way of displaying the feedback for each message
- Cleaned **A LOT** of unused code
- Added some translations

## Testing Instructions

The way to test this is basically append to the URL the flag wapuu, eg: `http://...?flags=wapuu`

1. Click on the ? icon in the master bar, located in the top right corner.
2. Click on Still need help?
3. A new and fancy chat should open
4. Test the behaviour of the following things
    - Jump to Recent button, works as desired (it appears when we are not in the bottom of the conversation, it takes you to the bottom of the conversation when clicked
    - Minimise icon will hide everything and nothing will remain visible (a part for the minimised Help Center bar) 
    - Typing long texts will move the Jump To Recent button upper (just type a few `shift-enter` characters)
5. The Send message feature is out of scope for this pull request, that's a follow up
6. Clear conversation and send transcript will be also a follow up pull request

See video for easier verification:
https://github.com/Automattic/wp-calypso/assets/5689927/b839f9a1-7fd7-4e16-b380-02062ab3b96e


## Pre-merge Checklist
